bundler/lib/bundler/lockfile_parser.rb

```diff
@@ -176,6 +212,31 @@ def parse_dependency(line)
       @dependencies[dep.name] = dep
     end
 
+    def parse_checksum(line)
+      return unless line =~ NAME_VERSION
+
+      spaces = $1
+      return unless spaces.size == 2
+      name = $2
+      version = $3
+      platform = $4
+      checksums = $6
+      return unless checksums
```

Comment:
nit: if this return is done before referencing `$2-4`, we can avoid allocating those strings in the short-circuit case